### PR TITLE
bump version to 1.9, migrate maintainers

### PR DESCRIPTION
**Pull request by henesy**
given marc seems to be awol, as title

### CODE REVIEW

**Comment by willysr** [comment]
@henesy same goes with this
please read https://slackbuilds.org/SBo-Github-Workflows.txt

**Comment by henesy** [comment]
@willysr rewrote commit message

**Comment by willysr** [comment]
@sbo-bot: build audio/xwax

File: audio/xwax/xwax.SlackBuild [hunk 1]
```diff
@@ -30,7 +31,7 @@
 cd $(dirname $0) ; CWD=$(pwd)
 
 PRGNAM=xwax
-VERSION=${VERSION:-1.6}
+VERSION=${VERSION:-1.9}
 BUILD=${BUILD:-2}
```
Comment (willysr) [inline audio/xwax/xwax.SlackBuild:35]: You need to reset BUILD into 1 if you update the VERSION

**Review by willysr — CHANGES_REQUESTED** [review]
please fix this

**Comment by henesy** [comment]
is it fine if the commits aren't squashed (as above?)

**Comment by willysr** [comment]
merged

**Pull request closed without merging**